connectd/usr/bin/connectd_startup_control

```diff
@@ -6,13 +6,13 @@
 #  installed remote.it attachment services for tcp listeners.
 #  This controls services registered using bulk and auto registration.
 #
-#  remot3.it, Inc. Copyright 2019. All rights reserved.
+#  remote.it, Inc. Copyright 2018. All rights reserved.
```

Comment:
This should have not changed. Legal name is `remot3.it, Inc.` and date should stay `2019`

Reply:
reverted to previous version of file